.github/issue_templates/Algorithim_development.md

```diff
@@ -0,0 +1,26 @@
+### Algorithm Description:
+
+### Requirements:
+
+> **Specify: (optional)**
+
+### Algorithm Code Info: Data/Input/Output
+
```

Comment:
We could even break down this section with subsections. Example, we could have subsection such as input data description, algorithm steps or pseudo code, output data descriptions.